panther_analysis_tool/main.py

```diff
@@ -1240,15 +1229,9 @@ def enrich_test_data(backend: BackendClient, args: argparse.Namespace) -> Tuple[
     all_relevant_specs = specs.detections + specs.simple_detections
 
     for spec in all_relevant_specs:
-        rule_id = spec.analysis_spec.get("RuleID", "")
-        if rule_id != "":
-            filtered_raw_analysis_items_by_id[rule_id] = raw_analysis_items_by_id[rule_id]
-            continue
-
-        policy_id = spec.analysis_spec.get("PolicyID", "")
-        if policy_id != "":
-            filtered_raw_analysis_items_by_id[policy_id] = raw_analysis_items_by_id[policy_id]
-            continue
+        analysis_id = lookup_analysis_id(spec.analysis_spec)
+        if analysis_id != UNKNOWN_ANALYSIS_ID:
+            filtered_raw_analysis_items_by_id[analysis_id] = raw_analysis_items_by_id[analysis_id]
```

Comment:
similar to above, do we want to log an info line here if the `analysis_id` is unknown?

Reply:
Updated